docs/datasources/global_client.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: "auth0"
+page_title: "Data Source: auth0_global_client"
+description: |-
+Retrieves a tenant's global Auth0 Application client
+---
+
+# Data Source: auth0_global_client
+
+Retrieves a tenant's global Auth0 Application client
+
+## Example Usage
+
+```hcl
+data "auth0_global_client" "global" {
+}
+```
+
+## Argument Reference
+
+No arguments accepted.
+
+## Attribute Reference
+
+* `client_id` - String. ID of the client.
+* `client_secret`<sup>[1](#client-keys)</sup> - String. Secret for the client; keep this private.
```

Comment:
@sergiughf At this point in time, I can't think a reasonable use case for exposing the client secret, but it's in line with the other client data source. What do you think about including?

Reply:
Considering we're using this as a data source to be ingested elsewhere I would tend to prefer removing it so we don't store it inside the state, just because it's a secret. If a valid use case for us having it there presents itself we could always re-add it. Would this work?

Reply:
Just thought I'd mention,  I use my custom branch that has this data source to keep different tenants custom login pages synced.  Specifically I have a master tenant that has my preferred custom login page and I use this data source to copy between tenants.

Reply:
@yinzara Really appreciate the input. After some investigation, that appears to be the primary use case for the global client DS. Am I correct in believing that you do not require the fetching of the client secret?